src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -145,6 +145,9 @@ def generate_info(self, commit_sha, coveralls_token, suite=None):
             if suite is None or suite in fname:
                 ordered_files.append('ccov-artifacts/' + fname)
 
+        r = requests.get('https://hg.mozilla.org/mozilla-central/json-rev/%s' % self.revision)
+        push_id = r.json()['pushid']
```

Comment:
Note: I assume that if the request fails, or if the output has an unexpected format (JSON or otherwise), this code will fail gracefully and explicitly point to what the exact problem was.